components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Engine/NetworkMessageEncoder.cs

```diff
@@ -170,23 +156,11 @@ public class NetworkMessageEncoder : IMessageEncoder {
                         encoder.WriteEncodeable(null, element);
                     }
                     encoder.Close();
-
-                    var body = Encoding.UTF8.GetBytes(writer.ToString());
-                    if (compressedPayload) {
-                        var stream = new MemoryStream(body);
-                        try {
-                            body = Encoding.UTF8.GetBytes(Convert.ToBase64String(stream.Zip()));
-                        }
-                        finally {
-                            stream.Dispose();
-                        }
-                    }
-
                     var encoded = new NetworkMessageModel {
-                        Body = body,
-                        ContentEncoding = compressedPayload ? "gzip" : "utf-8",
+                        Body = Encoding.UTF8.GetBytes(writer.ToString()),
+                        ContentEncoding = "utf-8",
                         Timestamp = DateTime.UtcNow,
-                        ContentType = compressedPayload ? ContentMimeType.Binary : ContentMimeType.UaJson,
+                        ContentType = ContentMimeType.Json,
```

Comment:
Shouldn't we have `ContentMimeType.UaJson` here?

Reply:
We are using `ContentMimeType.Json` in `main` branch so I decided to go with that. This change from `ContentMimeType.UaJson` to `ContentMimeType.Jso` has been introduced in #1562.